Spanner/src/Session/CacheSessionPool.php

```diff
@@ -643,31 +649,19 @@ private function getSession(array &$data)
      */
     private function createSessions($count)
     {
-        $args = [
+        $res = $this->database->connection()->batchCreateSessions([
```

Comment:
Also, does the PHP client use more than one gRPC channel? If so, we want to distribute the sessions as evenly as possible across the channels. No problem if only one channel is used in this implementation.

Reply:
PHP only uses a single gRPC channel per request lifecycle.